starwars/src/components/StarWars.css

```diff
@@ -1,14 +1,16 @@
 :root {
-
+  
```

Comment:
great on the style
![screencapture-14y483j6z3-codesandbox-io-2019-02-16-04_49_59](https://user-images.githubusercontent.com/37554037/52883555-64fac900-31a6-11e9-9ad3-7b705200b65d.png)